sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -37,11 +37,21 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     sqlContext.sql(s"DROP TABLE IF EXISTS jt")
   }
 
-  test("nested views") {
-    withView("jtv1", "jtv2") {
+  test("nested views (interleaved with temporary views)") {
+    withView("jtv1", "jtv2", "jtv3", "temp_jtv1", "temp_jtv2", "temp_jtv3") {
```

Comment:
@clockfly had helped verify that `DROP VIEW` actually handles temporary table, because internally `DROP VIEW` is translated to `DropTable`, which handles both persistent and temporary table, and temporary table is just temporary view. (OK, this is really confusing now...)